submissions/jeffersonMormah/htmlForm/contact.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
@Jeffbuchi Delete this folder and file

Reply:
@ToluAkin. Pls give a hint on your instruction so that I can understand. Thanks.